app/helpers/application_helper.rb

```diff
@@ -202,6 +202,7 @@ def view_to_url(view, parent = nil)
     case view.db
     when "AdvancedSetting"  then association = "advanced_settings"
     when "CloudNetwork"     then association = "cloud_networks"
+    when "OpenscapRuleResult"       then association = "openscap_rule_results"
```

Comment:
Note, we don't have to fix this here.

It's just that this case statement is hard to read.  Most of these string conversions are `view.db.tableize`.
It's not clear to me why some are not following that convention but even then, these exceptions can be handled on their own.